dashboard/db/migrate/20160913194550_add_hidden_stages.rb

```diff
@@ -0,0 +1,8 @@
+class AddHiddenStages < ActiveRecord::Migration[5.0]
+  def change
+    create_table :hidden_stages do |hidden_stage|
```

Comment:
Naively, I wouldn't expect `hidden_stages` to be associated with `sections` from the name.

Does it make sense to name it `hidden_section_stages` or the like?